automation_tools/__init__.py

```diff
@@ -1622,6 +1623,30 @@ def setenforce(mode):
     run('setenforce {0}'.format(mode))
 
 
+def host_pings(host):
```

Comment:
This function runs indefinitely if the pinged host never responds. How about a time-out? Please read `ping`'s man page and see if it has the requisite functionality built-in.

```
       Stop after sending count ECHO_REQUEST
              packets.  With  deadline  [ping(8)]  option,  ping waits for count ECHO_REPLY
              packets, until the timeout expires.
```